### DISCUSSION TITLE
[Config Support]: Motion detection values

### DISCUSSION THREAD

**Discussion by GrumpyMeow** [blakeblackshear/frigate#11415] [General]
A few weeks ago i did some tweaking of the motion-detection parameters i was using. I've managed to get a satisfactory result. But wouldn't it be very convenient to have some output of the output of the motion detector available?
Just like the audio-detector which publishes the output to MQTT.

**Comment by NickM-27** [comment]
There is a motion detected MQTT topic

**Comment by GrumpyMeow** [comment]
I did look through the MQTT topics, but only found a Boolean value. Or do i have to configure something to get more detailed values?

I’m more in search of the values which are compared to the defined motion-parameters. For example: the number of pixels which are above the threshold-parameter. And also the other parameters. 
It would allow for better fine tuning, i think.

**Reply by hawkeye217** [reply]
The upcoming version of Frigate has a motion tuner built into the UI that allows for real-time configuration of motion parameters (threshold/contour area/improve contrast).

But the docs describe the values you're looking for here:

https://docs.frigate.video/integrations/mqtt#frigatecamera_namemotion_thresholdset

**Comment by GrumpyMeow** [comment]
I've created a modification on my own fork, to publish values from the motion-detector to mqtt. 
This gives this kind of charts. I think this would help people to further tweak the motion detector.
Would this not be a nice-to-have feature?

![image](https://github.com/blakeblackshear/frigate/assets/12073499/e99d5bc1-43e0-409a-873a-5c31c726f019)

![image](https://github.com/blakeblackshear/frigate/assets/12073499/4e75c56b-fb75-45ce-aa4d-51b1e269568d)

**Reply by NickM-27** [reply]
Motion detection happens too often requiring updating the MQTT topic too much. and in general I think for motion visual is much more helpful in terms debugging and fine tuning. For now the new UI provides the tools we think are going to greatly improve the motion tuning experience.

**Reply by GrumpyMeow** [reply]
Ok. I understand. 
Thank you for this answer and the reasoning.
